comments-by-guideline-and-success-criterion.md

```diff
@@ -211,7 +211,7 @@ This applies directly as written, and as described in [Intent from Understanding
 
 <div class="note wcag2ict"> 
 
-The Intent section refers to the ability to allow users to enlarge the text on screen at least up to 200% without needing to use [assistive technologies](#dfn-assistive-technologies). This means that the application provides some means for enlarging the text 200% (zoom or otherwise) without loss of [content](#content-on-and-off-the-web) or functionality, or that the application works with the platform accessibility features to meet this success criterion.</div>
+The Intent section refers to the ability to allow users to enlarge the text on screen at least up to 200% without needing to use [assistive technologies](#dfn-assistive-technologies). This means that the application provides some means for enlarging the text 200% (zoom or otherwise) without loss of [content](#content-on-and-off-the-web) or functionality, or that the application works with the platform accessibility features to sasisfy this success criterion.</div>
```

Comment:
@maryjom this still seems to have sasisfy present vs. satisfy.

Reply:
Guess there were two occurrences of the same misspelling. Fixed the 2nd.